corehq/project_limits/rate_counter/rate_counter.py

```diff
@@ -54,26 +54,57 @@ def __init__(self, key, window_duration, window_offset=0, grains_per_window=1,
         )
 
     def get(self, scope, timestamp=None):
+        window_count_sum, _ = self._get_window_counts(scope, timestamp)
+        return window_count_sum
+
+    def get_count_and_wait_time(self, scope, threshold, timestamp=None):
+        # Why could this be negative?
+        if threshold < 0:
+            return 0, 0
```

Comment:
When using postman to my dev instance I observed that the scope `apiaccount:Dimagi Internal Test Account for Project <my local domain>` has a threshold of `-4900.0`.

Is this expected? Seems very odd.

Reply:
Still need to check it out, but do you think it makes sense to leave it in as a guard rail for now?